.gitmodules

```diff
@@ -11,6 +11,3 @@
 [submodule "vcpkg"]
 	path = vcpkg
 	url = https://github.com/microsoft/vcpkg.git
-[submodule "src/tests/resources/NonRegTests"]
```

Comment:
ok, I suppose that we get rid of non regression repository as a sub module because we don't play these tests yet / anymore